narayana-release-process.sh

```diff
@@ -78,8 +62,11 @@ then
     exit
   fi
   set -e
-  echo "Checking if there were any failed jobs, this may be interactive so please stand by"
-  JIRA_HOST=issues.redhat.com JENKINS_JOBS=60,60-AS_TESTS  ./scripts/release/pre_release.py
+  read -p "Checking if there were any failed CI jobs or blockers" NOBLOCKER
```

Comment:
This section should be updated to more exactly match https://github.com/jbosstm/narayana/commit/120febc4d0c7809faedbfbcdbf0a7545fc4918bc#diff-530cdb1e8542754f0d5c3b24ac1598f52957ed33177c8ca69b719666077c3bbeR59-R63

Reply:
Ah, it's my branch. I will adapt!